include/realtime_urdf_filter/urdf_filter.h

```diff
@@ -89,7 +89,7 @@ class RealtimeURDFFilter
     void render (const double* camera_projection_matrix, ros::Time timestamp = ros::Time());
 
     GLfloat* getMaskedDepth()
-      {return masked_depth_;}
+      {return (GLfloat*)masked_depth_.get();}
```

Comment:
This is not particularly nice, since it ruins the semantics of the unique pointer. I would recommend using cv::Mat to manage memory.